src/vs/workbench/api/browser/mainThreadChatCodeMapper.ts

```diff
@@ -56,8 +58,15 @@ export class MainThreadChatCodemapper extends Disposable implements MainThreadCo
 	$handleProgress(requestId: string, data: ICodeMapperProgressDto): Promise<void> {
 		const response = this._responseMap.get(requestId);
 		if (response) {
+			const edits = data.edits;
 			const resource = URI.revive(data.uri);
-			response.textEdit(resource, data.edits);
+			if (!edits.length) {
+				response.textEdit(resource, []);
```

Comment:
Hm... we also want to support empty edits (which signal start of "codemapping") for notebooks. This is important but we can keep is post PR